src/power_grid_model/validation/validation.py

```diff
@@ -149,10 +150,10 @@ def validate_batch_data(
     for batch, batch_update_data in enumerate(batch_data):
         row_update_data = compatibility_convert_row_columnar_dataset(batch_update_data, None, DatasetType.update)
         assert_valid_data_structure(row_update_data, DatasetType.update)
-        id_errors: list[ValidationError] = list(validate_ids_exist(row_update_data, input_data_copy))
+        id_errors: list[IdNotInDatasetError | InvalidIdError] = validate_ids(row_update_data, input_data_copy)
 
-        batch_errors = input_errors + id_errors
         if not id_errors:
+            batch_errors = input_errors
```

Comment:
```suggestion
        id_errors: list[IdNotInDatasetError | InvalidIdError] = validate_ids(row_update_data, input_data_copy)

        batch_errors = input_errors + id_errors
        if not id_errors:
```
Minor, but we should keep the original behaviour.

Reply:
What do you mean?

Reply:
Maybe first, what did you mean?

Reply:
Addressed.